handbook/engineering/search/product.md

```diff
@@ -28,6 +30,25 @@ It also owns a subset of features built on top of Sourcegraph search:
 
 ## Processes
 
+## Planning
+
+We work in two-week iterations.
+
+Iterations start every other Monday.
+
+On the last Thursday of an iteration:
```

Comment:
Calling out that this is set for Thursday and not Friday to allow for a full working day of async discussion. Iteration 0 does not comply 🙂